pom.xml

```diff
@@ -3,7 +3,7 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>ch.sbb</groupId>
-    <artifactId>export-package-generator</artifactId>
+    <artifactId>ch.sbb.export-package-generator</artifactId>
```

Comment:
@grigoriev I have changed the artifact id! is it ok?

Reply:
from my point of view it would be better leave it as it was.